packages/veto-app/shared/db/src/settings.ts

```diff
@@ -0,0 +1,27 @@
+import { IGunChainReference } from 'gun/types/chain'
+
+export type Settings = {
+  darkmode?: boolean
+  passwordConfirm?: boolean
+}
+
+const on = (gun: IGunChainReference<any>, callback: (key: string, value: any) => void) => {
```

Comment:
Since we only treat`Settings` here, it is safe to add its type to the function
```suggestion
const on = (gun: IGunChainReference<Settings, keyof Settings>, callback: (key: keyof Settings, value: Settings) => void) => {
```

Reply:
Updated